### ISSUE TITLE
Cannot invoke "Object.equals(Object)" because "results[0]" is null when attempting to call turtle.forward()

### ISSUE THREAD

**Issue by roobscoob** [cc-tweaked/cc-restitched#31]
the title plus the logs below describes the issue as much as possible. I don't know much about it yet.

In-game screenshot https://l.hall.ly/NTwG9

 - Minecraft version: 1.16.5
 - CC: Restitched version: v1.96.1-rc1
 - Logs: https://l.hall.ly/iS3Ib
 - Detailed reproduction steps: I have source code that reliably triggers this issue, but I can't seem to isolate what is actually triggering this (in the lua interpreter, running `turtle.forward()` works flawlessly). If possible I would like to refrain from uploading it.

**Comment by Merith-TK** [comment]
Can you share the code that *reliably* triggers this error?

**Comment by SquidDev** [comment]
Something like the following should work:

```lua
local co = coroutine.create(turtle.forward)
assert(coroutine.resume(co))
assert(coroutine.resume(co))
```

This should be fixed in 1.18 as it has the above commit merged.
